admproto/reader.go

```diff
@@ -24,10 +24,10 @@ func (r *Reader) Reset() {
 
 // Begin returns the header information out of the packet, and the remaining
 // data in the packet.
-func (r *Reader) Begin(in []byte) (out, application, instance_id []byte, err error) {
+func (r *Reader) Begin(in []byte) (out, application, instance_id []byte, numTags int, err error) {
```

Comment:
Can we use snake_case for consistency?

Reply:
My linter will continue to loudly complain, but yes.